spec/request/users_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET #index' do
+    it 'Response status is correct' do
+      get '/users'
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'Returns a successful response for index' do
+      get '/users'
+      expect(response).to be_successful
+    end
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=If%20a%20correct%20template%20was%20rendered.), This test should check If a correct template was rendered, not the condition of the response as you did. Please fix that.
```suggestion
    it 'Returns a successful response for index' do
      get '/users'
      expect(response).to render_template(:index)
    end
```